File: EASSET.Web/Modules/MasterData/KodeRekening/KodeRekeningForm.cs
using Serenity.ComponentModel;

namespace EASSET.MasterData.Forms;

[FormScript("MasterData.KodeRekening")]
[BasedOnRow(typeof(KodeRekeningRow), CheckNames = true)]
public class KodeRekeningForm
{
    public string Akun108Id { get; set; }
    public string AkunView { get; set; }
    public string Kelompok108Id { get; set; }
    public string KelompokView { get; set; }
    public string Jenis108Id { get; set; }
    public string JenisView { get; set; }
    public string Objek108Id { get; set; }
    public string ObjekView { get; set; }
    public string Ro108Id { get; set; }
    public string RoView { get; set; }
    public string SubRo108Id { get; set; }
    public string SubRoView { get; set; }
    public string SubSubRo108Id { get; set; }
    public string SubSubRoView { get; set; }
    public string KodeRekening108 { get; set; }
    public string ThAng { get; set; }
    public short IsEnabled { get; set; }
}

File: EASSET.Web/Modules/ServerTypes/MasterData/KodeRekeningForm.ts
import { PrefixedContext, initFormType } from "@serenity-is/corelib";
import { StringEditor, IntegerEditor } from "serenity.corelib";

export interface KodeRekeningForm {
    Akun108Id: StringEditor;
    AkunView: StringEditor;
    Kelompok108Id: StringEditor;
    KelompokView: StringEditor;
    Jenis108Id: StringEditor;
    JenisView: StringEditor;
    Objek108Id: StringEditor;
    ObjekView: StringEditor;
    Ro108Id: StringEditor;
    RoView: StringEditor;
    SubRo108Id: StringEditor;
    SubRoView: StringEditor;
    SubSubRo108Id: StringEditor;
    SubSubRoView: StringEditor;
    KodeRekening108: StringEditor;
    ThAng: StringEditor;
    IsEnabled: IntegerEditor;
}

export class KodeRekeningForm extends PrefixedContext {
    static readonly formKey = 'MasterData.KodeRekening';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!KodeRekeningForm.init)  {
            KodeRekeningForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;

            initFormType(KodeRekeningForm, [
                'Akun108Id', w0,
                'AkunView', w0,
                'Kelompok108Id', w0,
                'KelompokView', w0,
                'Jenis108Id', w0,
                'JenisView', w0,
                'Objek108Id', w0,
                'ObjekView', w0,
                'Ro108Id', w0,
                'RoView', w0,
                'SubRo108Id', w0,
                'SubRoView', w0,
                'SubSubRo108Id', w0,
                'SubSubRoView', w0,
                'KodeRekening108', w0,
                'ThAng', w0,
                'IsEnabled', w1
            ]);
        }
    }
}